custom_dc/README.md

```diff
@@ -0,0 +1,227 @@
+geoId/08,2021,10,10
+```
+
+The first column header is a property that identifies the observed entity,
+supported properties are `dcid`, `name`, `geoId`. When `dcid` is used, the
+entity should have been resolved (from a previous step).
+
+Refer to [sample folder](./sample) for supported example data files.
+
+Put all the input files under a local folder or Google Cloud Storage folder, and
+use the folder path for environment variable `SQL_DATA_PATH` as described in the
+Environment Variables section below.
+
+## Data Config
+
+A config file `config.json` is required and should be put under `SQL_DATA_PATH`.
+The detailed spec can be found in this
+[doc](https://github.com/datacommonsorg/import/blob/master/simple/stats/config.md).
+
+## UI Updates
+
+Default custom Data Commons use `custom` FLASK_ENV which uses a pre-defined html
+html and css customization. These customization are in the following folders:
+
+- html files: [location](../server/templates/custom_dc/custom/)
+- css and image files: [location](../static/custom_dc/custom/)
+
+Update these files for UI customization.
+
+## Local Development
+
+Custom Data Commons site via [localhost](http://localhost:8080) by following the
+instructions below:
+
+### Build docker image locally
+
+```bash
+docker build --tag datacommons-website-compose:latest \
+-f build/web_compose/Dockerfile \
+-t website-compose .
+```
+
+### Run custom Data Commons locally with SQLITE database
+
+In the root of this repository, run:
+
+```bash
+docker run -it \
+--env-file $PWD/custom_dc/sqlite_env.list \
+-p 8080:8080 \
+-v $PWD/custom_dc/sample:/sqlite \
+-v $PWD/server/templates/custom_dc:/workspace/server/templates/custom_dc \
+datacommons-website-compose:latest
+```
+
+## Run in Cloud
+
+Custom Data Commons can be ran on the cloud as a production service. The SQLite
+approach can still be used. Note you need to copy the data folders into the
+cloud disk and mount it to the docker container then specify the enviornment
+variables based on the requirements of the cloud providers.
+
+We have also provided a specific deployment setup on Google Cloud Run that is
+based on Cloud SQL.
+
+### Setup Cloud SQL
+
+Create a Cloud SQL instance from the [Cloud
+Console](https://console.cloud.google.com/sql/instances). Set instance ID as
+`dc-graph`, choose the type as "MySQL" and create a database `datacommons`. You
+will need to set a user and password. Record the instance connection name in the
+form of "<project>:<region>:dc-graph"
+
+### Setup Google Cloud Storage
+
+Google Cloud Storage is used to hold the data CSV files. From the [Cloud
+Console](https://console.cloud.google.com/storage/browser), create a new bucket
+or pick an existing bucket and upload the data CSV files there. It's recommended
+to create intermediate folders for the files for easier management.
+
+### Upload Data Files
+
+Upload the data CSV files into GCS and record the folder path as
+`gs://<bucket-name>/.../`.
+
+### Test Locally
```

Comment:
Suggest having a more clear title here. Perhaps "Run custom Data Commons locally with the CloudSQL database" to keep it consistent with the SQLITE section above?